src/event.js

```diff
@@ -36,7 +36,8 @@ export const customEvents = popup => {
 		getPreviewBody = ( element ) => {
 			return element.querySelector( '.wikipediapreview-body' ) ||
 				element.querySelector( '.wikipediapreview-disambiguation' ) ||
-				element.querySelector( '.wikipediapreview-error' )
+				element.querySelector( '.wikipediapreview-error' ) ||
+				element.querySelector( '.wikipediapreview-offline' )
```

Comment:
I really dislike this code. It's not you fault, and you're doing the right thing by adding the offline body class here but really, all states should just have the same `.wikipediapreview-body` element that can contain various things.

Reply:
Yea agree they should all have the same `wikipediapreview-body`, that will make the code more robust. This was evident when I added the offine preview card and saw it had broken `applyDragEvent`. Will take a look at how we can improve this.

Reply:
Updated with https://github.com/wikimedia/wikipedia-preview/pull/70/commits/21e24468bd76fece030d3037de97df86c6c43e28